geoindicators/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/geoindicators/TypologyClassification.groovy

```diff
@@ -152,11 +152,13 @@ IProcess identifyLczType() {
                                                                 THEN 105
                                                                 ELSE CASE WHEN ALL_VEGETATION<WATER_FRACTION_LCZ
                                                                         THEN 107
-                                                                        ELSE CASE WHEN HIGH_ALL_VEGETATION<0.1
-                                                                                THEN 104
-                                                                                ELSE CASE WHEN HIGH_ALL_VEGETATION<0.75
-                                                                                        THEN 102
-                                                                                        ELSE 101 END END END END AS LCZ1,
+                                                                        ELSE CASE WHEN HIGH_ALL_VEGETATION IS NULL
```

Comment:
Perhaps it 'd be more readable and easy to manage with a if else in groovy ?

Reply:
You mean using an "eachRow" and then test on eachRow ?

Reply:
yep